mcda/utils.py

```diff
@@ -0,0 +1,34 @@
+import pandas as pd
+import argparse
+import json
+
+
+
+def read_input_matrix(input_matrix_path: str) -> pd.DataFrame():
+    try:
+        with open(input_matrix_path, 'r') as fp:
+            return pd.read_csv(fp, sep=';')
+    except Exception as e:
+        print(e)
+
+
+
+#def save_result() ->:
```

Comment:
Unused code?

Reply:
Reminder for later: I'll add a "save result" function here eventually